src/commands/monthly.ts

```
@@ -25,7 +25,7 @@ export const monthlyCommand = define({
 		const monthlyData = await loadMonthlyUsageData({
 			since: ctx.values.since,
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)